### PR TITLE
Internal: Removed AI PLGs and New Connect flow [ED-15095]

### PR DESCRIPTION
**Pull request by DennisNerush**

### CODE REVIEW

File: modules/ai/assets/js/editor/ai-behavior.js [hunk 1]
```diff
@@ -124,40 +122,6 @@ export default class AiBehavior extends Marionette.Behavior {
 		return isDefaultValue ? this.getOption( 'buttonLabel' ) : this.getOption( 'editButtonLabel' );
 	}
 
-	getPromotionTexts( controlType ) {
```
Comment (DennisNerush) [inline modules/ai/assets/js/editor/ai-behavior.js:127]: Removing existing PLGs

File: modules/ai/assets/js/editor/module.js [hunk 2]
```diff
@@ -3,62 +3,34 @@ import { __ } from '@wordpress/i18n';
...
+									unmount();
+									rootElement.remove();
+								} } />
+						</LayoutAppWrapper>, rootElement );
+				}, 1000 );
 			}
 		} );
 	}
-
-	onCreateContainer( args ) {
```
Comment (DennisNerush) [inline modules/ai/assets/js/editor/module.js:18]: Removed PLG logic

File: modules/ai/assets/js/editor/pages/connect/connect-and-get-started.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+import { AIIcon } from '@elementor/icons';
+
+const ConnectAndGetStarted = ( { connectUrl, onSuccess, isConnected, getStartedAction } ) => {
+	const approveButtonRef = useRef();
+	const [ isTermsChecked, setIsTermsChecked ] = useState( false );
+
+	useEffect( () => {
+		// On local dev (as a standalone app), the connect lib is not loaded.
```
Comment (DennisNerush) [inline modules/ai/assets/js/editor/pages/connect/connect-and-get-started.js:12]: Same connect logic from

File: modules/ai/assets/js/editor/pages/form-layout/index.js [hunk 4]
```diff
@@ -69,7 +68,7 @@ const FormLayout = ( {
 	DialogContentProps = {},
 	attachments: initialAttachments,
 } ) => {
-	const { attachmentsTypes, onData, onInsert, onSelect, onClose, onGenerate, hasPro } = useConfig();
```
Comment (DennisNerush) [inline modules/ai/assets/js/editor/pages/form-layout/index.js:72]: Remove PRO upgrade indication as it's exists in the main layoyt

File: modules/announcements/triggers/ai-started.php [hunk 5]
```diff
@@ -16,7 +16,7 @@ class AiStarted extends Trigger_Base {
 	protected $name = 'ai-get-started-announcement';
 
 	public function after_triggered() {
-		User::set_introduction_viewed( [ 'introductionKey' => $this->name ] );
+		//User::set_introduction_viewed( [ 'introductionKey' => $this->name ] );
```
Comment (DennisNerush) [inline modules/announcements/triggers/ai-started.php:19]: Need to undo

File: modules/announcements/module.php [hunk 6]
```diff
@@ -94,28 +94,22 @@ public function enqueue_styles() {
...
+				'title' => 'Discover your new superpowers ',
+				'description' => '<p>With AI for text, code, image generation and editing, you can bring your vision to life faster than ever. Start your free trial now - <b>no credit card required!</b></p>',
 				'media' => [
 					'type' => 'image',
 					'src' => ELEMENTOR_ASSETS_URL . 'images/announcement.png?' . ELEMENTOR_VERSION,
 				],
 				'cta' => [
 					[
-						'label' => 'Continue',
+						'label' => 'Let\'s do it',
```
Comment (matipojo) [inline modules/announcements/module.php:105]:
```suggestion
						'label' => __( 'Let\'s do it', 'elementor' ),
```

File: modules/announcements/module.php [hunk 7]
```diff
@@ -94,28 +94,22 @@ public function enqueue_styles() {
...
 						'variant' => 'primary',
-						'target' => '_blank',
+						'target' => '_top',
+						'url' => '#welcome-ai',
 					],
 					[
-						'label' => 'Learn More',
-						'target' => '_blank',
-						'url' => 'https://go.elementor.com/whats-new-popup-learn-elementor-ai/',
+						'label' => 'Skip',
```
Comment (matipojo) [inline modules/announcements/module.php:111]: Translate

**Review by matipojo — DISMISSED** [review]

**Review by matipojo — DISMISSED** [review]

**Review by matipojo — APPROVED** [review]

**Pull request merged**